src/node.ts

```diff
@@ -61,7 +61,7 @@ function compile(code: string | any, filename: string) {
   delete opts.ignore;
   const output: swc.Output = swc.transformSync(code, {
     ...opts,
-    sourceMaps: opts.sourceMaps === undefined ? "inline" : opts.sourceMaps
+    sourceMaps: opts.sourceMaps === undefined ? true : opts.sourceMaps
```

Comment:
Babel uses `sourceMaps: 'both'` here, which is not supported by swc. So I change it to `true` to match the behavior of Babel.